lms/urls.py

```diff
@@ -427,6 +427,8 @@
         # Student Notes
         url(r'^courses/{}/edxnotes'.format(settings.COURSE_ID_PATTERN),
             include('edxnotes.urls'), name="edxnotes_endpoints"),
+
+        url(r'^api/v1/branding/', include('branding.api_urls')),
```

Comment:
I just realized that this URL is not consistent with our API conventions.  It should be `/api/branding/v1/` (version at the end).

@AlasdairSwan I'm going to change this URL, so we'll need to update the URL on Drupal as well.  Can't believe I didn't notice this until just now :\